cax/main.py

```diff
@@ -101,6 +101,10 @@ def main():
         corrections.AddElectronLifetime(),  # Add electron lifetime to run, which is just a function of calendar time
         corrections.AddGains(), #  Adds gains to a run, where this is computed using slow control information
         corrections.AddDriftVelocity(), #  Adds drift velocity to the run, also computed from slow control info
+	corrections.SetS2xyMap(),
```

Comment:
Did you fix it? Looks fine to me.

![huh](https://cloud.githubusercontent.com/assets/4679809/25919729/dc14e6a8-35cf-11e7-8ead-f544b0759e91.png)